### PR TITLE
Update definitions of isFsfLibre and isOsiApprovied values

### PR DESCRIPTION
**Pull request by goneall**
Resolves issue #212

### CODE REVIEW

File: model/ExpandedLicensing/Properties/isFsfLibre.md [hunk 1]
```diff
@@ -13,9 +13,9 @@ isFsfLibre specifies whether the [Free Software Foundation FSF](https://fsf.org)
 has listed this License as "free" in their commentary on licenses, located at
 the time of this writing at https://www.gnu.org/licenses/license-list.en.html.
 
-A value of "true" indicates that the FSF has listed this License as _free_.
+A value of "true"  the license is in the list of licenses that FSF publishes as libre.
```
Comment (kestewart) [inline model/ExpandedLicensing/Properties/isFsfLibre.md:16]: missing a verb - "indicates"?

File: model/ExpandedLicensing/Properties/isOsiApproved.md [hunk 2]
```diff
@@ -13,10 +13,9 @@ isOsiApproved specifies whether the [Open Source Initiative (OSI)](https://opens
 has listed this License as "approved" in their list of OSI Approved Licenses,
 located at the time of this writing at https://opensource.org/licenses/.
 
-A value of "true" indicates that the OSI has listed this License as approved.
+A value of "true" the license is in the list of licenses that OSI publishes as approvied.
```
Comment (kestewart) [inline model/ExpandedLicensing/Properties/isOsiApproved.md:16]: missing a verb - "indicates"?

**Review by kestewart — CHANGES_REQUESTED** [review]
Looks like a verb was missed in a couple of the comment updates

**Comment by goneall** [comment]
@kestewart Thanks for the catch - fixed

**Review by maxhbr — APPROVED** [review]
LGTM

**Comment by goneall** [comment]
@kestewart - if you could confirm it's all good, I'll merge

File: model/ExpandedLicensing/Properties/isOsiApproved.md [hunk 3]
```diff
@@ -13,10 +13,9 @@ isOsiApproved specifies whether the [Open Source Initiative (OSI)](https://opens
 has listed this License as "approved" in their list of OSI Approved Licenses,
 located at the time of this writing at https://opensource.org/licenses/.
 
-A value of "true" indicates that the OSI has listed this License as approved.
+A value of "true" indicates the license is in the list of licenses that OSI publishes as approvied.
```
Comment (swinslow) [inline model/ExpandedLicensing/Properties/isOsiApproved.md:16]: "approvied" => "approved"

File: model/ExpandedLicensing/Properties/isOsiApproved.md [hunk 4]
```diff
@@ -13,10 +13,9 @@ isOsiApproved specifies whether the [Open Source Initiative (OSI)](https://opens
...
 
-A value of "false" indicates that the OSI has not listed this License as
-approved.
+A value of "false" indicates the license is explicitly not in the corresponding list of OSI licenses (e.g. OSI publishes that a license is not approved).
```
Comment (swinslow) [inline model/ExpandedLicensing/Properties/isOsiApproved.md:18]: I might change the parenthetical to something like:

"(e.g. OSI has stated publicly that a license is not approved)"

**Review by swinslow — COMMENTED** [review]
Hi @goneall, thanks for this. One typo below and one suggested wording tweak, otherwise LGTM.

**Comment by goneall** [comment]
@kestewart @swinslow - I fixed the grammar and spelling errors - please take another look and if it looks good and approved, I'll merge.

File: model/ExpandedLicensing/Properties/isOsiApproved.md (hunk 2 above)
Comment (goneall) [reply model/ExpandedLicensing/Properties/isOsiApproved.md]: Fixed

File: model/ExpandedLicensing/Properties/isFsfLibre.md (hunk 1 above)
Comment (goneall) [reply model/ExpandedLicensing/Properties/isFsfLibre.md]: Fixed

File: model/ExpandedLicensing/Properties/isOsiApproved.md (hunk 3 above)
Comment (goneall) [reply model/ExpandedLicensing/Properties/isOsiApproved.md]: Fixed

File: model/ExpandedLicensing/Properties/isOsiApproved.md (hunk 4 above)
Comment (goneall) [reply model/ExpandedLicensing/Properties/isOsiApproved.md]: Added suggestion

**Review by swinslow — APPROVED** [review]

**Pull request merged**